site/content/en/docs/Getting started/first_kustomization.md

```diff
@@ -426,3 +426,396 @@ spec:
+
+replicas:
+- name: example-nginx
+  count: 2
+
+images:
+- name: nginx
+  newTag: latest
+
+labels:
+- pairs:
+    variant: var2
+  includeSelectors: false
+  includeTemplates: true
+
+resources:
+- ../../base
+
+patches:
+- patch-env-vars.yaml
+```
+
+_kustomize-example/overlays/staging/patch-env-vars.yaml_:
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: example-nginx
+spec:
+  template:
+    spec:
+      containers:
+      - name: nginx
+        env:
+        - name: ENVIRONMENT
+          value: stg
+```
+
+_Production overlay build_:
+
+```yaml
+$ kustomize build overlays/production/
+
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: nginx
+    variant: var1
+  name: env1-example-nginx-production
+  namespace: production
+spec:
+  ports:
+  - port: 80
+    protocol: TCP
+    targetPort: 80
+  selector:
+    app: nginx
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: nginx
+    variant: var1
+  name: env1-example-nginx-production
+  namespace: production
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+        variant: var1
+    spec:
+      containers:
+      - env:
+        - name: ENVIRONMENT
+          value: prod
+        image: nginx:1.20.2
+        name: nginx
+        ports:
+        - containerPort: 80
+
+```
+
+_Staging overlay build_:
+
+```yaml
+$ kustomize build overlays/staging/
+
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: nginx
+    variant: var2
+  name: env2-example-nginx-staging
+  namespace: staging
+spec:
+  ports:
+  - port: 80
+    protocol: TCP
+    targetPort: 80
+  selector:
+    app: nginx
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: nginx
+    variant: var2
+  name: env2-example-nginx-staging
+  namespace: staging
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+        variant: var2
+    spec:
+      containers:
+      - env:
+        - name: ENVIRONMENT
+          value: stg
+        image: nginx:latest
+        name: nginx
+        ports:
+        - containerPort: 80
+```
+
+### Next steps
+
+Congratulations on making it to the end of this tutorial.  As a summary for you, these are the customizations that were presented in this tutorial:
+
+- Add a name prefix and a name suffix
+- Set the namespace for your resources
+- Set the number of replicas for your deployment
+- Set the image to use
+- Add a label to your resources
+- Add an environment variable to a container by using a patch
+
+These are just a few of the things kustomize can do, and if you are interested to learn more the kustomization reference (to add link here) is your next step. You will see how you can use components to define base resources and add them to specific overlays where needed, use generators to create configMaps from files, and much more!
```

Comment:
```suggestion
These are just a few of the things kustomize can do. if you are interested to learn more, the kustomization reference (to add link here) is your next step. You will see how you can use components to define base resources and add them to specific overlays where needed, use generators to create configMaps from files, and much more!
```

Can we also add the link to the reference where it currently is now? Even though it has no content, it may well stay in that location, and having _some_ placeholder link should make it easier for us to update than a note would be.